backend/package.json

```diff
@@ -25,7 +32,11 @@
   "dependencies": {
     "express": "^4.17.1",
     "npm-run-all": "^4.1.5",
-    "rimraf": "^3.0.0"
+    "rimraf": "^3.0.0",
+    "cookie-parser": "~1.4.4",
+    "debug": "~2.6.9",
+    "http-errors": "~1.6.3",
+    "jade": "~1.11.0",
```

Comment:
jade?

Reply:
I removed these extra dependencies from package.json. 😀

Reply:
Did you resolve this? @harish-aka-shivi